### PR TITLE
totalSentiment() Method - Anna Pruett

### PR DESCRIPTION
**Pull request by ncannapruett**
This is my version of the totalSentiment method. I would love it if you could review it and make any changes. @nkeister8 @MalaZerb

### CODE REVIEW

File: Review.java [hunk 1]
```diff
@@ -165,4 +165,27 @@ public static String randomAdjective()
       return randomNegativeAdj();
     }
   }
+  public static double totalSentiment(String fileName)
```
Comment (MalaZerb) [inline Review.java:168]: Everything looks good! One thing that I would add is comments to the method

File: Review.java [hunk 2]
```diff
@@ -165,4 +165,27 @@ public static String randomAdjective()
       return randomNegativeAdj();
     }
   }
+  public static double totalSentiment(String fileName) 
+{
+   String file = textToString(fileName) String word = "";
+   String space = " ";
+   double totalVal = 0.0;
```
Comment (nkeister8) [inline Review.java]: I would maybe also add a double wordValue

**Review by MalaZerb — COMMENTED** [review]
Everything looks well! Onlt thing to add is comments to your method to make it easier to understand

**Comment by nkeister8** [comment]
This looks good! I would just clarify why you did some of the things you did in comments.

**Pull request merged**